### PR TITLE
adds persistence to the PtM process

### PR DESCRIPTION
**Pull request by allenjprice**
WIP - plz ping me before reviewing/merging

### CODE REVIEW

**Comment by allenjprice** [comment]
ok this is geting to an interesting place.

I have a re-frame->graphql mutation/resolver situation that appears narrow (only supports a handful of fields), but will get more and more interesting as we implement future steps.

would love to get some feedback at this point before continuing _too_ much further

File: src/cljs/apply/events.cljs [hunk 1]
```diff
@@ -13,7 +23,99 @@
...
+ (fn [{db :db} [_ response]]
+   (if-let [application (get-in response [:data :account :application])]
+     {:db (assoc db :application-id (:id application))
+      :dispatch [:app.init/somehow-figure-out-where-they-left-off application]}
+     {:dispatch [:app.init/create-application (get-in response [:data :account :id])]})))
+
+
+;;TODO
+(reg-event-fx
+ :app.init/somehow-figure-out-where-they-left-off
```
Comment (queenieho) [inline src/cljs/apply/events.cljs]: nice name 🥇

**Comment by allenjprice** [comment]
putting another hold on this one - for some reason i'm having some really sneaky trouble with in the resolvers. i think the problem is in the field resolvers for occupancy and term, but haven't been able to pinpoint it exactly.

**Comment by allenjprice** [comment]
the issue from yesterday has been cleared up; this is ready for FURTHER SCRUTINY

**Pull request merged**